### DISCUSSION TITLE
what is the appropriate type of RigidBody for bullets?

### DISCUSSION THREAD

**Discussion by SET001** [jcornaz/heron#281] [Q&A]
I need something that would be moved by a physical engine, that will collide with other objects but would not be affected by other objects' physics.
I was trying `RigidBody::KinematicVelocityBased` but with this, bullets are pushing objects they hit.

**Comment by jcornaz** [comment]
There is nothing that will provide you exactly that ouf-of-the box yet.

In the future, you will be able to set a lower dominance (#233) to the bullet, and higher dominance for other objects, so that the bullet do not move other objects.

In the meantime, you I see two possible solutions:
* access the underlying rapier bodies, and configure the dominance. (bodies are in the `RigidBodySet` resource, and can be accessed via the `RigidBodyHandle` component: `let body = rigid_body_set.get(handle.into_rapier())`)
* make the bullet a sensor and simulate the physics yourself for that bullet (pos += velocity * deltaTime; velocity += acceleration * deltaTime)
